libobs/util/platform-cocoa.m

```diff
@@ -386,6 +389,27 @@ uint64_t os_get_sys_free_size(void)
 	return vmstat.free_count * vm_page_size;
 }
 
+static void os_get_sys_total_size_internal()
+{
+	if (total_memory_initialized)
+		return;
+
+	total_memory_initialized = true;
+
+	size_t size;
+	int ret;
+
+	size = sizeof(total_memory);
+	ret = sysctlbyname("hw.memsize", &total_memory, &size, NULL, 0);
+}
+
+uint64_t os_get_sys_total_size(void)
+{
+	if (!total_memory_initialized)
+		os_get_sys_total_size_internal();
+	return total_memory;
```

Comment:
Prefer a blank line between the `if` body and the `return` statement, or use braces for the `if` and the `return` can go on the line immediately after the closing brace.

Reply:
For all of those: I kept the style here in line with the existing code:

https://github.com/obsproject/obs-studio/blob/4661c20c1810392f3d7448f14ae0898fd4e51d1a/libobs/util/platform-cocoa.m#L360-L365
https://github.com/obsproject/obs-studio/blob/4661c20c1810392f3d7448f14ae0898fd4e51d1a/libobs/util/platform-windows.c#L1289-L1295
https://github.com/obsproject/obs-studio/blob/4661c20c1810392f3d7448f14ae0898fd4e51d1a/libobs/util/platform-nix.c#L1045-L1051

Reply:
I also wonder if this is maybe a clang option we can set if that's a style we want to enforce?